pubsub/gcp/pubsub/pubsub.go

```diff
@@ -233,19 +233,19 @@ func (g *GCPPubSub) getPubSubClient(ctx context.Context, metadata *metadata) (*g
 }
 
 // Publish the topic to GCP Pubsub.
-func (g *GCPPubSub) Publish(req *pubsub.PublishRequest) error {
+func (g *GCPPubSub) Publish(ctx context.Context, req *pubsub.PublishRequest) error {
 	if !g.metadata.DisableEntityManagement {
-		err := g.ensureTopic(g.publishCtx, req.Topic)
+		err := g.ensureTopic(ctx, req.Topic)
```

Comment:
Ditto: please remove the now-unused `publishCtx` property

Reply:
Done.